src/libs/ajax/fetch/fetch-core.test.ts

```diff
@@ -0,0 +1,118 @@
+import { delay } from '@terra-ui-packages/core-utils';
+import { FetchFn } from '@terra-ui-packages/data-client-core';
+
+import { makeWithMaybeRetry, withErrorRejection, withMaybeRetry, withRetry } from './fetch-core';
+
+describe('withRetryOnError', () => {
```

Comment:
Change to `withMaybeRetry`.

Reply:
lol, oops... sweeping the test file for renames.